src/app/pages/account/update/index.js

```diff
@@ -0,0 +1,37 @@
+/* ============
+ * Account Update Page
+ * ============
+ *
+ * Page where the user can update the account information
+ */
+import accountService from '../../../services/account'
+import store from '../../../store/modules/account'
+
+
+export default {
+  computed: {
+    account: {
+      get () {
+        return store.state;
+      },
+      set (account) {
+        accountService.update(account);
+      }
+    }
+  },
+  methods: {
+    updateAccount(e) {
```

Comment:
Sorry for late response. I had problems with my PC.

Did you test this solution? For me when I change with your suggested changes and add `console.log(account)` inside this method I receive original object, not changed with form.